src/connectors/hits/__tests__/connectHits-test.js

```diff
@@ -217,10 +217,14 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hits/js/#co
       createURL: () => '#',
     });
 
+    const expectedHits = [{ name: 'transformed' }, { name: 'transformed' }];
+    expectedHits.__escaped = true;
+    results.hits.__escaped = true;
```

Comment:
What's this for? I believe this is unused.

Reply:
Best expects arrays to have extra properties as well, do fails without this

Reply:
I tried and it doesn't fail without this.

Reply:
ah yes, it's not needed because the reference is the same